### PR TITLE
Queue fixes

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: src/main/java/org/jitsi/utils/queue/QueueStatistics.java [hunk 1]
```diff
@@ -118,6 +94,9 @@ public JSONObject getInstanceStats()
         stats.put("add_rate", addRate.getRate(now));
         stats.put("remove_rate", removeRate.getRate(now));
         stats.put("drop_rate", dropRate.getRate(now));
+        double duration = (now - firstPacketAddedMs) / 1000;
```
Comment (bbaldino) [inline src/main/java/org/jitsi/utils/queue/QueueStatistics.java]: should this be `/ 1000.0`?

File: src/main/java/org/jitsi/utils/queue/PacketQueue.java [hunk 2]
```diff
@@ -124,10 +102,10 @@ public static boolean logDroppedPacket(int numDroppedPackets)
     private final int capacity;
 
     /**
-     * The number of packets which were dropped from this {@link PacketQueue} as
-     * a result of a packet being added while the queue is at full capacity.
+     * Handles dropped packets and exceptions thrown while processing.
      */
-    private final AtomicInteger numDroppedPackets = new AtomicInteger();
+    @NotNull
+    private ErrorHandler errorHandler = new ErrorHandler(){};
```
Comment (bbaldino) [inline src/main/java/org/jitsi/utils/queue/PacketQueue.java]: where's the hook to override this (say with `CountingErrorHandler`)?

Comment (bbaldino) [reply src/main/java/org/jitsi/utils/queue/PacketQueue.java]: oops, sorry, see it now.

**Comment by bgrozev** [comment]
I'll fix the double

**Pull request merged**